source/UF/H-Levels.lagda

```diff
@@ -0,0 +1,163 @@
+Martin Escardo and Ian Ray, 06/02/2024
```

Comment:
This should be "Ian Ray" only, I think.

Reply:
I copied some of your proofs from the original file :)